packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -288,12 +288,21 @@ class _DropdownMenuState<T> extends State<_DropdownMenu<T>> {
             textStyle: route.style,
             child: ScrollConfiguration(
               behavior: const _DropdownScrollBehavior(),
-              child: Scrollbar(
-                child: ListView(
-                  controller: widget.route.scrollController,
-                  padding: kMaterialListPadding,
-                  shrinkWrap: true,
-                  children: children,
+              child: PrimaryScrollController(
```

Comment:
Is there a reason for the PrimaryScrollController instead of just passing the scroll controller into the Scrollbar? Nothing wrong with it, just curious about the choice here.

Reply:
Because both Scrollbar and ListView need to set controllers, it looks more intuitive to use PrimaryScrollController.

Reply:
Makes sense, thanks!